test/parallel/test-zlib-failed-init.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 
 const assert = require('assert');
 const zlib = require('zlib');
 
+if (process.config.variables.node_shared_zlib && /^1\.2\.[0-8]$/.test(process.versions.zlib)) {
```

Comment:
Nit: long line. I think linter is not happy with it.

Reply:
Hmm... from digging through the Makefile this appears to be the lint command:
```
$(NODE) tools/eslint/bin/eslint.js --cache --rulesdir=tools/eslint-rules --ext=.js,.md benchmark doc lib test tools
```

I ran that and it didn't show any lint in that file. I'll break the long line anyways, however.

Reply:
Thank you.